### DISCUSSION TITLE
Separate button than AccordionButton to close the Accordion Panel

### DISCUSSION THREAD

**Discussion by Razakhan45** [chakra-ui/chakra-ui#8823] [Ideas] [answered]
### Description

I have created an accordion and when i open an accordion panel by click on top accordion button then i am hiding accordion button and only showing accordion panel . To again close that accordion Panel i have created another Accordion Button inside the panel on which i click to close the panel but this why defaultIndex or index value of accordion does not work properly because of 2 accordion button in one accordion item. 
As opening and closing of a panel depends on mapping of the button first button close first panel and so on ..
now this is the problem add this as a feature to create button to close a panel which does not effect the main Accordion button to close and open. the panel.
or suggest me a why to handle this :
i cannot create an <AccordionButton> tag but i want to have the ability to close the panel by clicking on a custom html element.

### Problem Statement/Justification

1. add flexibility to close panel click somewhere else like showing an cross icon inside the panel .
2. I am mapping email threads with the accordion close panel have toggle button visible in which i show message title , some one body message and then left aligned time . but when panel opens it show complete email details username , time, title , body. In that case accordion button is showing the same thing which is kind of unnecessary and looks bad. 

### Proposed Solution or API

There are two button so now first panel open in defaultindex is given 1 instead of 0
<Accordion  defaultIndex={[selectedMessage.length - 1 ]} allowMultiple>
          {selectedMessage.map((selectedMessage, index , array) => (
            <>
                <AccordionItem border={"none"}>
                  <h2 style={{display: array.length == 1? "none":""}} >
                    <AccordionButton justifyContent={"space-between"} padding={"20px"} gap={"8px"} _hover={{backgroundColor:"transparent"}} alignItems={"center"}>
                      <Box gap={'24px'} display={"flex"} flexGrow={1} overflow={"hidden"}  whiteSpace={"nowrap"} textAlign="left">
                         <Text flexShrink={0} whiteSpace={"nowrap"} overflow={"hidden"} textOverflow={"ellipsis"} className={Styles.name}>{selectedMessage?.from.split("<")[0].trim()}</Text>
                         <Text flexShrink={1} whiteSpace={"nowrap"} overflow={"hidden"} textOverflow={"ellipsis"} className={Styles.message}>{selectedMessage?.snippet}</Text>
                      </Box>
                     <Text className={Styles.date}>{ProperFormat(formatDate(selectedMessage.date))}</Text>
                    </AccordionButton>
                  </h2>
                  <AccordionPanel border={array.length == 1? "none":"1px solid #E9E9F0"} borderRadius={"16px"} margin={array.length == 1 ?"20px 16px":"0px 24px 24px 24px"}>
                    <Box justifyContent={"space-between"} display={"flex"} margin={"4px 0px 24px 0px"} alignItems={"center"}>
                    <AccordionButton display={"flex"} gap={"8px"} alignItems={"center"}>
                    <Avatar
                    name={selectedMessage?.from.split("<")[0].trim()}
                    height={"35px"}
                    width={"35px"}
                    src={""}
                  ></Avatar> 
                  <Text className={Styles.heading}>{selectedMessage?.from.split("<")[0].trim()}</Text>
                    </AccordionButton>
                    <Box display={"flex"} gap={"8px"} alignItems={"center"}>
                      <Text className={Styles.date} marginRight={"16px"}>{!isToday(selectedMessage.date)
  ? `${formatTimeEmail(selectedMessage.date)} (${ProperFormat(formatDate(selectedMessage.date))})`
  : formatTimeEmail(selectedMessage.date)}</Text>
                      <EmailLeft cursor={"pointer"}/>
                      <EmailRight cursor={"pointer"}/>
                    </Box>
                    </Box>
                    // alot of my code comes the data to show in the panel  then close panel , Accordion item and outside mapping closing tag of Accordion

**Comment by segunadebayo** [comment, marked as answer]
Use `Accordion.ItemTrigger` for extra close controls, or control `value`.
